src/v2/ion/uiSchemaTransformer.js

```diff
@@ -181,77 +143,79 @@ const createUISchema = (transformedResp, remediationForm) => {
     .flatten()
     .value();
 
-  const getCheckboxUiSchema = ({ label, type, required }) => ({
-    // For Remember Me checkbox, we need the label only on the right side of it.
-    placeholder: label,
-    label: false,
-    // Separating prop type for Backbone.Model
-    // from html input type
-    modelType: type,
-    // uiSchema type is the html input type desired.
-    type: 'checkbox',
-    required: required || false,
-  });
-
-  const getPasswordUiSchema = () => ({
-    type: 'password',
-    params: {
-      showPasswordToggle: true,
-    },
-  });
-
-  const getFactorsUiSchema = ({ options }, factors) => ({
-    type: 'factorSelect',
-    options: createFactorTypeOptions(options, factors),
-  });
-
-  const getAuthenticatorsUiSchema = ({ options }, authenticators) => ({
-    type: 'authenticatorSelect',
-    options: createAuthenticatorOptions(options, authenticators),
-  });
-
   return remediationValue.map(ionFormField => {
     const uiSchema = {
       'label-top': true,
-      type: 'text',
     };
 
-    if (ionFormField.type === 'boolean') {
-      Object.assign(uiSchema, getCheckboxUiSchema(ionFormField));
-    }
+    if (ionFormField.type === 'string' || !ionFormField.type) {
+      uiSchema.type = 'text';
 
-    if (ionFormField.secret === true) {
-      Object.assign(uiSchema, getPasswordUiSchema());
-    }
-
-    if (Array.isArray(ionFormField.options)) {
-      uiSchema.type = 'radio';
-    }
+      // { name: 'password', secret: true }
+      if (ionFormField.secret === true) {
+        Object.assign(uiSchema, getPasswordUiSchema());
+      }
 
-    // select factor form for multiple factor enroll and multiple factor verify
-    // when factor has not been enrolled we get back factorProfileId, and once its enrolled
-    // we get back factorId
-    if (ionFormField.name === 'factorId' ||
-      ionFormField.name === 'factorProfileId') {
-      const factors = transformedResp.factors && transformedResp.factors.value || [];
-      Object.assign(uiSchema, getFactorsUiSchema(ionFormField, factors));
+      //
+      if (Array.isArray(ionFormField.options)) {
+        if (ionFormField.name === 'factorId'
+          || ionFormField.name === 'factorProfileId') {
+          // { name: 'factorId' | 'factorProfileId', type: 'string', options: [ {label: 'xxx', value: 'yyy'} ]}
+          //
```

Comment:
nit: no comments

Reply:
It was intentional to separate example with comment. will remove it.